fcm/fcm.go

```diff
@@ -127,7 +138,7 @@ func (f *FCM) ConnectToFcm(fcmNotifications chan<- notification.PrinterNotificat
 					}
 				} else {
 					// stop listening unknown error happened.
-					log.Errorf("%v,%v", err1, err2)
+					log.Errorf("Unexpected error happened on FCM listener: %v, %v", err1, err2)
```

Comment:
If you can go down to one error, this will be cleaner, too.

Reply:
Fixed.